Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -86,23 +87,31 @@ public IssuesClientTests()
     }
 
     [IntegrationTest]
-    public async Task CanCreateRetrieveAndCloseIssue()
+    public async Task CanCreateAssignRetrieveAndCloseIssue()
     {
         var newIssue = new NewIssue("a test issue") { Body = "A new unassigned issue" };
+        newIssue.Labels.Add("test");
+        newIssue.Assignees.Add(_context.RepositoryOwner);
+
         var issue = await _issuesClient.Create(_context.RepositoryOwner, _context.RepositoryName, newIssue);
         try
         {
             Assert.NotNull(issue);
+            Assert.True(issue.Assignees.All(x => x.Login == _context.RepositoryOwner));
 
             var retrieved = await _issuesClient.Get(_context.RepositoryOwner, _context.RepositoryName, issue.Number);
             var all = await _issuesClient.GetAllForRepository(_context.RepositoryOwner, _context.RepositoryName);
             Assert.NotNull(retrieved);
-            Assert.True(all.Any(i => i.Number == retrieved.Number));
+            Assert.True(retrieved.Assignees.Count == 1);
+            Assert.True(retrieved.Assignees[0].Login == _context.RepositoryOwner);
+            Assert.True(all.Any(i => i.Number == retrieved.Number && i.Assignees.Count == 1 && i.Assignees[0].Login == _context.RepositoryOwner));
```

Comment:
:lipstick: we're fetching both and doing a mix of assertions here:

- could we order this so we set and assert `retrieved`, and then set and assert `all`?
- could we use multiple `Assert.True` expressions here, rather than one big one sprinkled with `&&`s?